py-polars/polars/dataframe/frame.py

```diff
@@ -9485,6 +9485,10 @@ def hash_rows(
         seed_3
             Random seed parameter. Defaults to `seed` if not set.
 
+        .. note::
+            This implementation of :func:`hash_rows` does not guarantee stable results
+            across Polars versions. It only has guaranteed stability within one version.
+
```

Comment:
```suggestion
            This implementation of :func:`hash_rows` does not guarantee consistent results across different Polars versions. Its stability is only guaranteed within a single version.

```
I'd like to suggest this rephrasing for better clarity. What do you think?